doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/index.md

```diff
@@ -4,4 +4,71 @@ title: C
 permalink: /:path/
 sort_idx: 10
 ---
-Using C Kaa endpoint SDK
+
+- [Intruduction](#introduction)
+- [Build configuration](#build-configuration)
+
+# Introduction
+
+*C SDK* was designed to be the best fit for the embedded devices. Small footprint, modularity, low memory consumption, portability with the C's execution speed make C SDK the best candidate for using it even with low-power microcontrollers. Using C SDK requires no operation system, so you can use it on barebones systems.
+
+# Build configuration
+
+C SDK uses CMake as a build system. So, build configuration is performed on the make file generation stage. To pass any configuration parameter, it should be prefixed with `-D`, e.g.:
+
+    cmake -DCMAKE_INSTALL_PREFIX=~/username/installdir
+
+The available configuration parameters list:
+
+1. **KAA_MAX_LOG_LEVEL** - Maximum log level used by C SDK. The higher value the more detailed logs.
+
+    Values:
+
+        0 - NONE (no logs)
+        1 - FATAL
+        2 - ERROR
+        3 - WARN
+        4 - INFO
+        5 - DEBUG
+        6 - TRACE
+
+        Default: If build type is `Release`, **KAA_MAX_LOG_LEVEL=4**. If `Debug`, **KAA_MAX_LOG_LEVEL=6**
```

Comment:
According to the documentation style *italic* should be used (https://github.com/kaaproject/kaa/pull/724/files#diff-ce3d7e7f43897801e9d5df519e9b82abR48). Will fix.